<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionSaveAs_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut_2"/>
    <addaction name="actionCopy_2"/>
    <addaction name="actionPaste_2"/>
    <addaction name="separator"/>
    <addaction name="actionUndo_2"/>
    <addaction name="actionRedo_2"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="separator"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionDark_Light"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/open_light.svg</normaloff>:/tooldbar_images/open_light.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconText">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/save_light.svg</normaloff>:/tooldbar_images/save_light.svg</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="iconText">
    <string>Save...</string>
   </property>
   <property name="toolTip">
    <string>Save the file on local disk</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normalon>:/tooldbar_images/zcut.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Cut...</string>
   </property>
   <property name="toolTip">
    <string>Cut the selected text</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/copy_light.svg</normaloff>:/tooldbar_images/copy_light.svg</iconset>
   </property>
   <property name="text">
    <string>Copy...</string>
   </property>
   <property name="toolTip">
    <string>Copy the selected text</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/paste_light.svg</normaloff>:/tooldbar_images/paste_light.svg</iconset>
   </property>
   <property name="text">
    <string>Paste...</string>
   </property>
   <property name="toolTip">
    <string>Paste text on the clipboard</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/undo_light.svg</normaloff>:/tooldbar_images/undo_light.svg</iconset>
   </property>
   <property name="text">
    <string>Undo...</string>
   </property>
   <property name="toolTip">
    <string>Undo previous action</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/redo_light.svg</normaloff>:/tooldbar_images/redo_light.svg</iconset>
   </property>
   <property name="text">
    <string>Redo...</string>
   </property>
   <property name="toolTip">
    <string>Redo the last undone action</string>
   </property>
  </action>
  <action name="actionDark_Light">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/mode_light.png</normaloff>
     <normalon>:/tooldbar_images/mode_dark.png</normalon>:/tooldbar_images/mode_light.png</iconset>
   </property>
   <property name="text">
    <string>Dark Mode...</string>
   </property>
   <property name="iconText">
    <string>Dark/Light...</string>
   </property>
   <property name="toolTip">
    <string>Toggle between dark and light themes</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
  </action>
  <action name="actionSaveAs_2">
   <property name="text">
    <string>S&amp;ave as...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionCut_2">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy_2">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste_2">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionColored_text">
   <property name="text">
    <string>Colored text</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/about_light.svg</normaloff>
     <normalon>:/tooldbar_images/about_dark.svg</normalon>:/tooldbar_images/about_light.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About QtPad</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources/toolbar_resources.qrc">
     <normaloff>:/tooldbar_images/save_as.svg</normaloff>:/tooldbar_images/save_as.svg</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save file as</string>
   </property>
  </action>
  <action name="actionUndo_2">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo_2">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/toolbar_resources.qrc"/>
 </resources>
 <connections/>
</ui>
